flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/io/BarrierBuffer.java

```diff
@@ -123,11 +123,11 @@
 	 * <p>There is no limit to how much data may be buffered during an alignment.
 	 *
 	 * @param inputGate The input gate to draw the buffers and events from.
-	 * @param bufferBlocker The buffer blocker to hold the buffers and events for channels with barrier.
+	 * @param bufferStorage The buffer blocker to hold the buffers and events for channels with barrier.
```

Comment:
Nit: The Description of bufferStorage in Javadoc should be modified accordingly.